java8/src/main/java/io/cucumber/java8/Java8DocStringTypeDefinition.java

```diff
@@ -0,0 +1,28 @@
+package io.cucumber.java8;
+
+import io.cucumber.core.backend.DocStringTypeDefinition;
+import io.cucumber.docstring.DocStringType;
+import net.jodah.typetools.TypeResolver;
+
+final class Java8DocStringTypeDefinition extends AbstractGlueDefinition implements DocStringTypeDefinition {
+
+    private final DocStringType docStringType;
+
+    @Override
+    public DocStringType docStringType() {
+        return docStringType;
+    }
+
+    Java8DocStringTypeDefinition(Object body, String contentType) {
+        super(body, new Exception().getStackTrace()[3]);
+        Class returnType = TypeResolver.resolveRawArguments(DocStringDefinitionBody.class, body.getClass())[0];
+        this.docStringType = new DocStringType(
+            this.method.getReturnType(),
+            contentType.isEmpty() ? method.getName() : contentType,
```

Comment:
This was copied from `JavaDocStringTypeDefinition` but I don't think it makes sense here. Lambdas have no meaningfull method name so the content type is always mandatory. This should throw an error when content type is empty or null.